src/main/ts/components/Editor.tsx

```diff
@@ -30,9 +22,10 @@ export interface IProps {
   toolbar: NonNullable<EditorOptions['toolbar']>;
   disabled: boolean;
   textareaName: string;
-  tinymceScriptSrc: string;
+  tinymceScriptSrc: string | string[] | ScriptItem[];
   rollback: number | false;
   scriptLoading: {
+    block?: boolean;
     async?: boolean;
     defer?: boolean;
     delay?: number;
```

Comment:
Kind of redundant to have this and script items?

Reply:
So you propose we just use an empty list to `tinymceScriptSrc`? I guess that could work.

Reply:
I just noticed that if I provide an empty list to `loadList` it will never call either of the callbacks.

Reply:
Ok, fixed so an empty array to `tinymceScriptSrc` does what the `block` would do. Also an empty array to `loadList` is now considered a failure (maybe I should just throw an error?) though that code path should not be used.